src/cargo/ops/cargo_rustc/context.rs

```diff
@@ -1326,13 +1326,17 @@ fn add_target_specific_suffixes(
         ret.push((".wasm".to_string(), TargetFileType::Normal, true));
     }
 
-    // rust-lang/cargo#4490
-    //  - only uplift *.dSYM for binaries.
+    // rust-lang/cargo#4490, rust-lang/cargo#4960
+    //  - only uplift debuginfo for binaries.
```

Comment:
I am not sure this is exactly what we want.

If you crate is a 

```
[lib]
crate-type = [ "cdylib" ]
```

you might actually be interested in debuginfo as well?

Reply:
True yeah, but I think it's fine to always fix that in a follow up (or when it arises)